src/test/java/seedu/address/logic/commands/AddCommandTest.java

```diff
@@ -103,36 +105,71 @@ public Path getAddressBookFilePath() {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
+        public Path getCalendarFilePath() {
+            throw new AssertionError("This method should not be called.");
+        }
+
```

Comment:
Consider adding this method and the test methods below to a separate class called <code>AddEventCommandTest.java</code> to separate tests for Bangyi's <code>AddCommand</code> and tests for your <code>AddEventCommand</code>.

Reply:
This method is for a Model stub, model interface now has a getCalendarFilePath() method that needs to be implemented for the stub to work for AddCommandTest class. Not implementing this will cause compilation error.